Xamarin.PropertyEditing/ViewModels/BrushPropertyViewModel.cs

```diff
@@ -176,12 +198,16 @@ private void SetBrushType (CommonBrushType type)
 				Value = null;
 				break;
 			case CommonBrushType.Solid:
+			case CommonBrushType.MaterialDesign:
```

Comment:
This seems like it adds several changes in behavior to the material type when the brush type is changed, but I don't see a test covering the change. You're changing the value, committing, and then changing the color _again_ via `SetToClosest`.